Dockerfile.rhel

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2017-present Sonatype, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM registry.access.redhat.com/rhel7/rhel
+
+MAINTAINER Sonatype <cloud-ops@sonatype.com>
+
+LABEL name="Nexus IQ Server image" \
+  vendor=Sonatype \
+  version="1.84.0-01" \
+  release="1.84.0" \
+  url="https://www.sonatype.com" \
+  summary="The Nexus IQ Server" \
+  description="Nexus IQ Server is a policy engine powered by precise intelligence on open source components. \
+    It provides a number of tools to improve component usage in your software supply chain, allowing you to \
+    automate your processes and achieve accelerated speed to delivery while also increasing product quality"
+  com.sonatype.license="Apache License, Version 2.0" \
+  com.sonatype.name="Nexus IQ Server image" \
+  run="docker run -d -p 8070:8070 -p 8071:8071 IMAGE" \
+  com.sonatype.license="Apache License, Version 2.0" \
+  com.sonatype.name="Nexus IQ Server base image" \
+  io.k8s.description="Nexus IQ Server is a policy engine powered by precise intelligence on open source components. \
+    It provides a number of tools to improve component usage in your software supply chain, allowing you to \
+    automate your processes and achieve accelerated speed to delivery while also increasing product quality"
+    com.sonatype.license="Apache License, Version 2.0" \
+  io.k8s.display-name="Nexus IQ Server" \
+  io.openshift.expose-services="8071:8071" \
+  io.openshift.tags="Sonatype,Nexus,IQ Server"
+
+# Optional parameters.
+ARG IQ_SERVER_VERSION=1.84.0-01
+ARG IQ_SERVER_SHA256=81e877f4e793e740cd0be38faaa35650448bfcb77aa30fd48d15babeb01a9a09
+
+# Mandatory parameters. Docker needs to know volume mount point and location of startup script.
+ENV SONATYPE_WORK="/sonatype-work" \
+    IQ_HOME="/opt/sonatype/nexus-iq-server/" \
+    DOCKER_TYPE="rh-docker"
+
+ARG IQ_SERVER_COOKBOOK_VERSION="release-0.4.20200210-175005.e554f17"
+# ARG IQ_SERVER_COOKBOOK_URL="https://github.com/sonatype/chef-nexus-iq-server/releases/download/${IQ_SERVER_COOKBOOK_VERSION}/chef-nexus-iq-server.tar.gz"
+ARG IQ_SERVER_COOKBOOK_URL="/home/ec2-user/chef-nexus-iq-server.tar.gz"
```

Comment:
i think that's not right ...

Reply:
indeed, looks like the remote url was commented out during testing?

Reply:
Indeed. Thanks guys - that was a testing thing (the arg wasn't getting picked up like I expected).